docs/guides/migrating-from-camunda-platform-7/adjusting-bpmn-models.md

```diff
@@ -0,0 +1,199 @@
+---
+id: adjusting-bpmn-models
+title: Adjusting BPMN models
+description: "Learn how to adjust your BPMN models when migrating from Camunda Platform 7 to Camunda Platform 8."
+---
+
+Ensure your BPMN process models are adjusted as follows to migrate them from Camunda Platform 7 to Camunda Platform 8:
+
+- The namespace of extensions has changed from `http://camunda.org/schema/1.0/bpmn` to `http://camunda.org/schema/zeebe/1.0`.
+- Different configuration attributes are used
+- Camunda Platform 8 has a _different coverage_ of BPMN elements (see [Camunda Platform 8 BPMN coverage](/components/modeler/bpmn/bpmn-coverage.md) versus [Camunda Platform 7 BPMN coverage](https://docs.camunda.org/manual/latest/reference/bpmn20/)), which might require some model changes. Note that the coverage of Camunda Platform 8 will increase over time.
+
+The following sections describe what the existing community-supported [backend diagram converter](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/backend-diagram-converter) does by BPMN symbol and explain unsupported attributes.
+
+### General considerations
+
+The following attributes/elements **cannot** be migrated:
+
+- `camunda:asyncBefore`: Every task in Zeebe is always asyncBefore and asyncAfter.
+- `camunda:asyncAfter`: Every task in Zeebe is always asyncBefore and asyncAfter.
+- `camunda:exclusive`: Jobs are always exclusive in Zeebe.
+- `camunda:jobPriority`: There is no way to prioritize jobs in Zeebe (yet).
+- `camunda:failedJobRetryTimeCycle`: You cannot yet configure the retry time cycle. Alternatively, you can [modify your code](/apis-tools/grpc.md#input-failjobrequest) to use the `retryBackOff` timeout (in ms) for the next retry.
+
+### Service tasks
+
+Migrating a service task is described in detail in the section about adjusting your source code below:
```

Comment:
What section about adjusting source code are we referring to exactly here?

Reply:
This was missing some context, so I added it to an admonition. 

> :::note
Migrating a service task is described in [adjusting your source code](/guides/migrating-from-camunda-platform-7/adjusting-source-code.md). You will have both BPMN and source code changes.
:::